addons/im_livechat/static/src/core/common/thread_model_patch.js

```diff
@@ -2,20 +2,15 @@
 
 import { Record } from "@mail/core/common/record";
 import { Thread } from "@mail/core/common/thread_model";
+import { assignDefined } from "@mail/utils/common/misc";
 
 import { patch } from "@web/core/utils/patch";
 
 patch(Thread, {
     _insert(data) {
         const thread = super._insert(...arguments);
         if (thread.type === "livechat") {
```

Comment:
condition not really necessary, if the `operator` is present we should handle it, and otherwise it makes no difference